src/pages-and-resources/flexible_peer_grading/Settings.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
+
+import AppSettingsModal from '../app-settings-modal/AppSettingsModal';
+import messages from './messages';
+
+const OpenResponsesSettings = ({ intl, onClose }) => (
+  <AppSettingsModal
+    appId="flexible_peer_grading"
```

Comment:
Are we unable to use `useIntl` here?

Reply:
this is an `appId`. It is being use to identify with the `appId` from `edx-platform` https://github.com/openedx/edx-platform/pull/32589/files#diff-0e47f76ce6dbfbf2727a7950c8e784078b1ab6451988ffec4021af06a479d893R277

Reply:
lol, I meant in the file in general. vs injectIntl

Reply:
I think the problem is larger than that. I couldn't get `useIntl` to work with small amount of work so I decide not to pursue it. It would make unit test so much easier.

Reply:
For context, it seems these file are load using `React.lazy` and somehow doesn't have the right provider for useIntl